### DISCUSSION TITLE
Wikiのページ構造がどうなってるかわかる人居ますか…

### DISCUSSION THREAD

**Discussion by A-B-S-I** [freeserverproject/freeserver-wiki#112] [Q&A] [answered]
いろいろな記事が個別で書かれていってるのは把握しているのですが、  
トップページから全てのページに飛べる構造になっているのか、私自身が把握できていないのです。  
どなたか把握していらっしゃいませんか？

**Comment by Nakatai-0322** [comment, marked as answer]
はい。

トップページにすべてのページが表示されているのではなく、トップページはただの他記事と同じ編集できるファイルです。（index.md）

トップページにリンクがぺたぺた貼り付けられてWikiとして成り立つわけです。

**Reply by A-B-S-I** [reply]
現時点でどれがつながってないとか全部把握してます？
してなければ、こちらでリストアップしてしまうのですが…

**Reply by Nakatai-0322** [reply]
全部は把握してないですね...

ミスでつなげてない記事も一つ二つぐらいはありそうです。

**Reply by A-B-S-I** [reply]
んー、今色々見ているんですが…
player-postでディレクトリ独立している構造だと、分類するときに問題が生じそうです。
中にある椛谷の記事、さっき私が作った、main-areaのディレクトリ(ブランチ参照)に移動させて、問題ないです？

**Reply by Nakatai-0322** [reply]
多分問題ないかと。

**Comment by A-B-S-I** [comment]
小さな話だが、資源系とエキストラ系のワールド、ディレクトリ切ってしまわん？という話。

**Reply by Nakatai-0322** [reply]
確かに。肥大化しそうですね。
